extension/_locales/en/messages.json

```diff
@@ -0,0 +1,82 @@
+{
+    "extensionName": {
+      "message": "Firefox Translations",
+      "description": "Name of the extension"
+    },
+    "extensionDescription": {
+      "message": "Translate websites in your browser without using the cloud.",
+      "description": "Description of the extension."
+    },
+    "loadingTranslationEngine": {
+      "message": "Loading Translation Engine",
```

Comment:
Should this have ellipsis at the end, since it's a `loading…` message?

Reply:
We can add ellipsis.